lib/bungie_client.dart

```diff
@@ -274,16 +274,16 @@ class BungieClient {
             .map((skull) => skull['displayName']));
   }
 
-  /// Returns the progress on Moments of Triumph as a completion percentage for
+  /// Returns the progress on Age of Triumph as a completion percentage for
```

Comment:
Alright, then the right way to do this seems to be to have two routes - TriumphAgeHandler, and TriumphMomentsHandler. Then build handlers for each and call bungie_client getAge or getMoments.

I'll take a look at this probably this weekend.